src/tracing/CMakeLists.txt

```diff
@@ -43,9 +43,9 @@ endfunction()
 
 set(osd_traces oprequest.tp osd.tp pg.tp)
 add_tracing_library(osd_tp "${osd_traces}" 1.0.0)
-add_tracing_library(rados_tp librados.tp 2.0.0)
+add_tracing_library(librados_tp librados.tp 2.0.0)
```

Comment:
> This fix works because this line?

sorry, i don't follow you. which fix? and why is the output **name** of a library related to the build failure? 

> it's weird that I can not find ibrados_tp library (the same for other *_tp libraries). According to

`make rados-tp`, and try again. or with your change `make librados_tp`.

> there's a another build error following, so "librados_tp" target has not been started to build, right?


with your change, `librados_tp` does not depends on build_rocksdb.